opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -184,6 +220,12 @@ public SpanBuilder asChildOf(Span parent) {
         }
 
         @Override
+        public SpanBuilder ignoreActiveSpan() {
+            firstParent = null;
```

Comment:
This should be handled differently, currently it also clears explicitly added references.